linchpin/api/fetch/fetch_git.py

```diff
@@ -0,0 +1,48 @@
+import os
+import subprocess
+import tempfile
+
+from fetch import Fetch
+from linchpin.exceptions import LinchpinError
+
+
+class FetchGit(Fetch):
+    def __init__(self, ctx, fetch_type, src, dest, cache_dir, root):
+        super(FetchGit, self).__init__(ctx, fetch_type, dest, root)
+        self.src = src
+
+        self.cache_dir = os.path.join(cache_dir, "git")
+        if not os.path.exists(self.cache_dir):
+            os.mkdir(self.cache_dir)
+
+    def fetch_files(self):
+        # key cannot contain ':' since linchpin does not support python 3
```

Comment:
Can you better explain this comment? We are targeting python 3 in the near-ish future and I'd like to avoid statements like this one.